.github/workflows/cherrypick.yml

```diff
@@ -0,0 +1,71 @@
+name: Create Cherrypick PR
+
+on:
+  pull_request:
+      types:
+        - closed
+      branches:
+        # TODO extract this to an env variable?
+        - 'master'
+
+env:
+  # TODO add a way to handle multiple potential cherrypick targets.
+  TARGET_BRANCH: '4.3'
+  USERNAME: 'Godot Organization'
+  EMAIL: 'noreply@godotengine.org'
+
+jobs:
+  Create-cherrypick-PR:
+    # The cherrypick label is hardcoded because contains() doesn't seem to be able to use an environment variable as a second argument.
```

Comment:
```suggestion
    # The cherrypick label is hardcoded because `contains()` doesn't seem to be able to use an environment variable as a second argument.
```